duckduckgo/iOS
# Bookmarks in autocomplete

Task/Issue URL:
- https://app.asana.com/0/856498667320406/1198906255639949
- https://app.asana.com/0/856498667320406/1198963987362725
Tech Design URL:
CC:

**Description**:
Surface bookmarks in autocomplete. Improve UX around suggestion-based searched and when external keyboard is attached.

**Steps to test this PR**:
1. Add some bookmarks/favorites.
2. Search in autocomplete - if search is relevant up to two local results should appear.

Additional cases:
- Small iPhone, iPad when autocomplete is obscured by keyboard list should be scrollable.
- Using arrow keys to navigate suggestions.
- Using suggestion button from autocomplete.



**Copy Testing**:


**Orientation Testing**:


**Device Testing**:


**OS Testing**:


**Theme Testing**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
